task_5.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+
+"""
+The program adds a vector to each row of a given matrix.
+"""
+
+if __name__ == '__main__':
+    matrix = np.array([[1, 2, 3], [4, 5, 6]])
+    newRow = np.array([1, 1, 1])
+    for row in range(matrix.shape[0]):
+        matrix[row] += newRow
```

Comment:
The beauty of numpy is that in most cases for loops are not needed (and are usually less efficient), try using broadcasting.
https://numpy.org/doc/stable/user/basics.broadcasting.html